Update Hotjar destination docs (#3137)
segmentio/segment-docs: src/connections/destinations/catalog/hotjar/index.md

rewrite: true
id: 5913371070a3e552b9561a4e
---
[Hotjar](https://hotjar.com) is the fast and visual way to really understand your users, offering a full set of tools: analytics, page heatmaps, session recordings, feedback tools, and more. Hotjar gives you everything your team needs to uncover user insights and make the right product changes.

The Segment Hotjar Destination allows you to easily get started with Hotjar and its core APIs:

> 1. Automatically install the [Hotjar Tracking Code](https://help.hotjar.com/hc/en-us/articles/115011639927);
> 2. Automatically send [user attributes](https://help.hotjar.com/hc/en-us/articles/360033640653-Identify-API-Reference) to Hotjar by connecting your Segment `identify` calls with Hotjar's own Identify API;
> 3. Automatically send [custom events](https://help.hotjar.com/hc/en-us/articles/4405109971095-Events-API-Reference) to Hotjar by connecting your Segment `track` calls with Hotjar's own Events API.

Reviewer: ```suggestion
1. Automatically install the [Hotjar Tracking Code](https://help.hotjar.com/hc/en-us/articles/115011639927){:target="_blank"}.
2. Automatically send [user attributes](https://help.hotjar.com/hc/en-us/articles/360033640653-Identify-API-Reference){:target="_blank"} to Hotjar by connecting your Segment `identify` calls with Hotjar's own Identify API.
3. Automatically send [custom events](https://help.hotjar.com/hc/en-us/articles/4405109971095-Events-API-Reference){:target="_blank"} to Hotjar by connecting your Segment `track` calls with Hotjar's own Events API.
```

> Knowing who your users are and what they're doing unlocks much more advanced filtering and targeting capabilities across all of Hotjar's tools, helping you find meaningful insights much faster.

Reviewer: ```suggestion
Knowing who your users are and what they're doing unlocks more advanced filtering and targeting capabilities across all of Hotjar's tools, helping you find meaningful insights faster.
```

## Getting Started

{% include content/connection-modes.md %}

> 1. From the Segment web app, click **Catalog**;

Reviewer: ```suggestion
1. Navigate to **Connections** and click **Add Destination** From the Segment web app.
```

> 2. Search for "Hotjar" in the Catalog, select it, and choose which of your Javascript sources to connect the destination to;

Reviewer: ```suggestion
2. Search for *Hotjar* in the Catalog, select it, and choose the Javascript source you want to connect the destination to.
```

> 3. Add your **Hotjar Site ID** to your Destination settings. You can find this ID (e.g. 123456) in Account settings > Sites & Organizations;

Reviewer: ```suggestion
3. Add your **Hotjar Site ID** to your Destination settings. You can find this ID in Account settings > Sites & Organizations.
```

4. Your changes will appear in the Segment CDN in about 45 minutes, and then Analytics.js will start asynchronously loading Hotjar's tracking snippet and sending data. If you are already using Hotjar, remove Hotjar's snippet from your code.

## Identify

> The Hotjar destination will automatically ingest a User ID and any values sent over your Identify spec as [traits](/docs/connections/spec/identify/#traits), as long as session capture and user attributes are both enabled in Hotjar.

Reviewer: ```suggestion
The Hotjar destination automatically ingests a User ID and any values sent over your Identify spec as [traits](/docs/connections/spec/identify/#traits), as long as session capture and user attributes are both enabled in Hotjar.
```

Identify calls that do not have a User ID value will not be sent to Hotjar.

### Nested values or lists

> As of July 2022, the Hotjar Identify API does not yet support ingesting values passed as nested objects or lists over your identify Spec:

Reviewer: Do we need to call out the date? Can't we just have it say: 
"The Hotjar Identify API doesn't support ingesting values passed as nested objects or lists over your identify Spec."

Author: Updated the copy

    }
```

> In the example above, Hotjar would reject all the values in the `address` field.

Reviewer: ```suggestion
In the example above, Hotjar rejects all the values in the `address` field.
```

## Track

> The Hotjar destination will automatically ingest any user actions tracked over your Track spec as [events](/docs/connections/spec/track/), as long as session capture is enabled in Hotjar.

Reviewer: ```suggestion
The Hotjar destination automatically ingests any user actions tracked over your Track spec as [events](/docs/connections/spec/track/), as long as session capture is enabled in Hotjar.
```

### Event properties

> As of July 2022, the Hotjar Events API does not yet support ingesting event properties:

Author: What about something like "Currently, the Hotjar Events API doesn't support ingesting event properties:"

But don't feel strongly about it
